packages/react-core/src/demos/ComposableMenu/examples/FavoritesDemo.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { MenuToggle, Divider, Dropdown, DropdownGroup, DropdownList, DropdownItem } from '@patternfly/react-core';
+
+export const FavoritesDemo: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [favorites, setFavorites] = React.useState<string[]>([]);
+  const menuRef = React.useRef<HTMLDivElement>(null);
+
+  const onToggleClick = (ev: React.MouseEvent) => {
+    ev.stopPropagation();
+    setTimeout(() => {
+      if (menuRef.current) {
+        const firstElement = menuRef.current.querySelector('li > button:not(:disabled)');
+        firstElement && (firstElement as HTMLElement).focus();
+      }
+    }, 0);
+    setIsOpen(!isOpen);
+  };
```

Comment:
```suggestion
  const onToggleClick = (ev: React.MouseEvent) => {
    setIsOpen(!isOpen);
  };
```

Similar to above comments